src/events.py

```
@@ -1,23 +1,30 @@
 import random
 
+from parso.python.tree import String
```

Ruff (0.8.2)

`parso.python.tree.String` imported but unused

Remove unused import: `parso.python.tree.String`

(F401)